src/modules/ROOT/pages/changelog/202303/00.json

```diff
@@ -0,0 +1,800 @@
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/751/commits/b4d5c7fd7cdef17c847717e2944fc669c213204d"
+      ]
+    },
+    {
+      "what": [
+        "ability to serve data as response in an integration gateway route."
+      ],
+      "why": [
+        "to give an endpoint response greater extensibility to integration gateway as different data can be returned."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/775/commits/b250b9329fc05bc6da2f65fb8632b3b2bda937dc",
+        "https://github.com/cortezaproject/corteza/pull/775/commits/921d4988c75159bc25a6296255327d8c99c762f8",
+        "https://github.com/cortezaproject/corteza/pull/775/commits/6c423084b336d35367c733457d41a138f2d2d5c0",
+        "https://github.com/cortezaproject/corteza/pull/775/commits/31ca5d6c297ec1569f01d72a9389a5590e3486b9",
+        "https://github.com/cortezaproject/corteza/pull/775/commits/11dccaea9f299895e5637929c02e1b0dfb518bcd"
+      ]
+    },
+    {
+      "what": [
+        "Route description field, Filters sorting, Profiler hits purging and Integration gateway settings."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/775/commits/a64f4c18b3d428bb207ab9f7f95c5a42324a9bab",
+        "https://github.com/cortezaproject/corteza/pull/775/commits/352865b4fb9aab5d0a1697bca88987f06d4e913e"
+      ]
+    },
+    {
+      "what": [
+        "support for viewing and undeleting a deleted single or multiple records in a record list."
+      ],
+      "why": [
+        "to allow users to view and restore deleted records."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/740/commits/101c1c7e3026285052c9a731684c2cb278047144"
+      ]
+    },
+    {
+      "what": [
+        "`BETWEEN` and `NOT BETWEEN` operator support for single value fields (number and date-time module fields)."
+      ],
+      "why": [
+        "to extend record filtering support for range filtering of numeric and date-time for single value field."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/813/commits/2027409b0d8875a48fd0017264ba03f5535733ec"
+      ]
+    },
+    {
+      "what": [
+        "support for a fully customizable navigation page block with multiple navigation item type and customizable options for the navigation list items."
+      ],
+      "why": [
+        "to have the ability to show one or two level of dynamic and static navigation."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/41f1283e9d393393c651b8ac5a28cae09733f1fd"
+      ]
+    },
+    {
+      "what": [
+        "ability to drill down charts, by clicking on a data entry inside the chart.",
+        "Based on the configuration it can either filter an existing record list on the page, that is showing the records from the same module as the chart, or open a new record list in a modal."
```

Comment:
```suggestion
        "Based on the configuration it can either filter an existing record list on the page that is showing the records from the same module as the chart or open a new record list in a modal."
```